docs/pages/platform/upgrading/2.2.mdx

```diff
@@ -0,0 +1,49 @@
+---
+meta:
+  title: "Upgrading to 2.2"
+  parentTitle: "Upgrading"
+  description: "Guide to upgrade to Liveblocks version 2.2"
+---
+
+We are making Thread "resolved" a first-class citizen property. You don't have
```

Comment:
```suggestion
We are making “resolved”  a first-class citizen property on [Thread](...). You don't have
```
My computers broken, and I'm on mobile. Its difficult to make changes, could you link to the thread component in react-ui api reference? Also replace all straight quotes on the page with curly ones (e.g. '  to ’). Thanks!

Reply:
Done!